ratatool-sampling/src/main/java/com/spotify/ratatool/GcsConfiguration.java

```diff
@@ -82,15 +82,15 @@ public static Configuration get() {
 
           conf.setIfUnset(
               HadoopCredentialConfiguration.BASE_KEY_PREFIX +
-                  HadoopCredentialConfiguration.ENABLE_SERVICE_ACCOUNTS_SUFFIX,
+              HadoopCredentialConfiguration.ENABLE_SERVICE_ACCOUNTS_SUFFIX,
```

Comment:
wondering if we can get rid of this class. 
Seems like it is only being used in `ParquetIO` (besides AvroSampler and an example)

Reply:
Yea, Parquet cleanup is not a priority for me at the moment but I think we can get rid of it once that's done.